app/domain/person/filter/attributes.rb

```diff
@@ -100,10 +112,22 @@ def sql_value(value, constraint)
     when 'match', 'not_match'
       then "%#{ActiveRecord::Base.send(:sanitize_sql_like, value.to_s.strip)}%"
     when 'equal', 'greater', 'smaller' then value
+    when 'in' then sql_array_value(value)
     else raise("unexpected constraint: #{constraint.inspect}")
     end
   end
 
+  def sql_array_value(value)
+    return "('')" unless value.present? && value.is_a?(Array)
+
+    sql_array = if value.first.is_a? String
+                  "'#{value.join('\',\'')}'" # Strings have to be in quotes ('a','b')
+                else
+                  value.join(',')
+                end
+    "(#{sql_array})"
```

Comment:
Bitte setze niemals mit String Operationen einen User-kontrollierten Wert in einen SQL Query ein. Unbedingt parametrisiere Queries (mit Fragezeichen) verwenden.